packages/typespec-client-generator-core/src/decorators.ts

```diff
@@ -586,7 +586,7 @@ export function listOperationsInOperationGroup(
     for (const op of current.operations.values()) {
       // Skip templated operations
       if (!isTemplateDeclarationOrInstance(op)) {
-        operations.push(getOverriddenClientMethod(context, op) ?? op);
+        operations.push(op);
```

Comment:
is this the decided way to proceed until every language can fully support overrides? I feel it would be better to still return these instead of the original, and we could tell emitter authors that it's overridden, and we can point back to the original operation as well

Reply:
`listOperationsInOperationGroup` is a decorator helper, which return the raw typespec operation. if you return the override one, then http payload will be changed. it is a bug, not a design change.

Reply:
ahh i see, approving